app/assets/javascripts/discourse/app/components/uppy-image-uploader.js

```diff
@@ -35,6 +35,9 @@ export default class UppyImageUploader extends Component {
       type: this.type,
       additionalParams: this.additionalParams,
       validateUploadedFilesOptions: { imagesOnly: true },
+      uploadDropTargetOptions: () => ({
+        target: document.querySelector(".uploaded-image-preview"),
```

Comment:
Would there be a problem using a class name if there are multiple uppys on the same page?

Reply:
hmmm yes that's a good questions 🤔 need to check, and think of how to fix it if that's a problem :+1: